jest.setup.js

```diff
@@ -1 +1 @@
-jest.setTimeout(10000)
+jest.setTimeout(30000)
```

Comment:
Do we need to leave this as 30000?

Reply:
I don't think this ended up being the root cause of the issue we were facing so can probably be restored.